src/DnsUtils.ts

```diff
@@ -0,0 +1,53 @@
+import DnsRecordsError, { DnsRecordsErrorCode } from "./errors/dnsRecordsError";
+import { CryptoRecords, DnsRecord, DnsRecordType } from "./publicTypes";
+import { isStringArray } from "./utils";
+
+export default class DnsUtils {
+
+  public toClassical(record: CryptoRecords): DnsRecord[] {
+    const dnsTypes = this.getAllDnsTypes(record);
+    return ([] as DnsRecord[]).concat(
+      ...dnsTypes.map(type => this.constructDnsRecords(record, type))
+    );
+  }
+
+  public toCrypto(records: DnsRecord[]): CryptoRecords {
+    const cryptoRecords:CryptoRecords = {};
+    for (const record of records) {
+      const { ttl, type, value } = record;
+      
+      const dnsRecord = cryptoRecords[`dns.${type}`];
+      const ttlInRecord = cryptoRecords[`dns.${type}.ttl`] as number | undefined;
+
+      if (isStringArray(dnsRecord)) {
+        dnsRecord.push(value);
+      } else {
+        cryptoRecords[`dns.${type}`] = [value];
```

Comment:
You may have arrays in this funciton internally, but you need to `JSON.stringify` arrays before returning.